app/src/main/res/drawable/ic_star.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
i think you can use https://material.io/icons/#ic_star_rate icon instead of adding this star.xml file,